src/main/kotlin/cacophony/controlflow/FunctionHandler.kt

```diff
@@ -52,7 +102,19 @@ class FunctionHandlerImpl(
         TODO("Not yet implemented")
     }
 
-    private fun introduceStaticLinksParams(): List<CFGNode> {
-        TODO("Not yet implemented")
+    override fun getStaticLink(): Variable.AuxVariable.StaticLinkVariable {
+        return staticLink
+    }
+
+    // Creates staticLink auxVariable in analyzedFunction, therefore shouldn't be called multiple times.
+    // Static link is created even if parent doesn't exist.
+    private fun introduceStaticLinksParams() {
+        // I truly don't know who is responsible for layout of the stack
+        // TODO: uncomment (and fix) after #127 is merged
+//            registerVariable(
+//                staticLink,
+//                VariableAllocation.OnStack(/*what should be there?*/),
```

Comment:
I think you can allocate it in a register, later register coloring will likely put it on stack